src/webui/www/webui.qrc

```diff
@@ -7,6 +7,7 @@
         <file>private/confirmruleclear.html</file>
         <file>private/confirmruledeletion.html</file>
         <file>private/css/Core.css</file>
+        <file>private/css/palette.css</file>
```

Comment:
Wrong order. The list is sorted alphabetically.
Also you are missing an entry for public palette.css.